uint8 amount
---
rosnp_msgs/ROSNumpyList_UInt16 depth_imgs
